### PR TITLE
Clean up outcome printer test infra

### PR DESCRIPTION
**Pull request by chenglou**
It's just another snapshot. We'll reuse the previous rock solid snapshot "infra" aka `git diff`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -70,19 +70,18 @@ benchmarks/refmt_main3b.cmx: benchmarks/refmt_main3b.ml
...
-	./node_modules/.bin/reanalyze -all-cmt . -suppress tests -suppress benchmarks
-	./test.sh
+test: reanalyze build-native lib/test.exe
 	./lib/test.exe
+	./test.sh
 
-roundtrip-test: bootstrap lib/test.exe
-	./node_modules/.bin/reanalyze -all-cmt . -suppress tests -suppress benchmarks
-	ROUNDTRIP_TEST=1 ./test.sh
+roundtrip-test: reanalyze bootstrap lib/test.exe
```
Comment (IwanKaramazow) [inline Makefile:77]: does the `-suppress benchmarks` still matter?

File: tests/res_diff.ml [hunk 2]
```diff
@@ -1,128 +0,0 @@
-type change = {
```
Comment (IwanKaramazow) [inline tests/res_diff.ml:1]: RIP :+1:

**Review by IwanKaramazow — APPROVED** [review]
image

File: Makefile (hunk 1 above)
Comment (chenglou) [reply Makefile:77]: Removed

**Pull request merged**